pkgs/development/python-modules/NSKeyedUnArchiver/default.nix

```
@@ -0,0 +1,26 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  lib,
+  pythonOlder,
+}:
+
+buildPythonPackage rec {
+  pname = "NSKeyedUnArchiver";
```

See https://nixos.org/manual/nixpkgs/stable/#contributing-guidelines

```suggestion
  pname = "nskeyedunarchiver";
```